warehouse/query-core/src/main/java/datawave/query/jexl/functions/EvaluationPhaseFilterFunctions.java

```diff
@@ -316,6 +316,34 @@ public static boolean excludeRegex(Iterable<?> values, String regex) {
         return includeRegex(fieldValue, regex);
     }
     
+    // Evaluate text. Note this is being done against the un-normalized value.
+    public static FunctionalSet<ValueTuple> includeText(Object fieldValue, String valueToMatch) {
+        FunctionalSet<ValueTuple> matches = FunctionalSet.emptySet();
+        if (fieldValue != null && ValueTuple.getStringValue(fieldValue).equals(valueToMatch)) {
```

Comment:
getStringValue returns the value as stored, correct?

Reply:
yes